### DISCUSSION TITLE
2.0.3

### DISCUSSION THREAD

**Discussion by baimard** [baimard/gestion#140] [Announcements]
I never modify again database in my life.

Hotfix bug in database that cause to destroy data; ex siret and siren column

## What's Changed
* Hotfix 2.0.3 by @baimard in https://github.com/baimard/gestion/pull/139

**Full Changelog**: https://github.com/baimard/gestion/compare/2.0.2...2.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/baimard/gestion/releases/tag/2.0.3'>2.0.3</a>.</em>

**Comment by Valdnet** [comment]
The application version has not been changed in the `info.xml` file.

**Reply by baimard** [reply]
Yes ... I know my apologizes for that.
